lib/src/init.dart

```diff
@@ -47,6 +47,10 @@ DynamicLibrary initRealm() {
     }
 
     if (Platform.isLinux) {
+      if (isFlutterPlatform) {
+	return '${File(Platform.resolvedExecutable).parent.path}/lib/lib$binaryName.so';
```

Comment:
I have the feeling this won't work if the dart file is compiled. We don't do this on the other places. We just use the current directory. So I think we better do the same here. If we are sure that the binary is placed in `lib` then lets return that instead.
`currentdir/lib/librealm_dart.so`

Reply:
but I have not tested that

Reply:
I only do it for flutter, which is compiled. In my tests, this is where the lib gets placed.